src/Stripe.Tests/transfers/when_creating_a_transfer_with_application_fee.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ***** This test needs a bank account setup in order to run correctly, and the ignore attribute wasn't working. Good times.
```

Comment:
Ha! I feel ya on the universal "Good times" frustration - I recently created a TestSources class that will give you valid bank account options. You can just create the bank account service and pass that. Also, for testing just the application fee, you don't need to worry about it having the behaviors (the protected types at the top and the behaves_like call). All that can be skipped - we just need to assert this one thing. I've been moving away from behaviors lately, they seem to confuse people - confirm? :)

Thanks for your work - this is great.